tests/integration_tests/bugs/test_lp1900837.py

```diff
@@ -0,0 +1,28 @@
+"""Integration test for LP: #1900836.
+
+This test mirrors the reproducing steps from the reported bug: it changes the
+permissions on cloud-init.log to 600 and confirms that they remain 600 after a
+reboot.
+"""
+import pytest
+
+
+def _get_log_perms(client):
+    return client.execute("stat -c %a /var/log/cloud-init.log")
+
+
+@pytest.mark.sru_2020_11
```

Comment:
bikeshed alert, I'm not sure what dates get us? major_minor version of SRU 20_4? This comment conflicts with my next comment